exporter/importables.go

```diff
@@ -556,7 +585,8 @@ var resourcesMap map[string]importable = map[string]importable{
 				defaultValue, dok := policy["defaultValue"]
 				typ := policy["type"]
 				if !vok && !dok {
-					log.Printf("[INFO] Skipping policy element as it doesn't have both value and defaultValue")
+					log.Printf("[DEBUG] Skipping policy element as it doesn't have both value and defaultValue. k='%v', policy='%v'",
```

Comment:
If the above line is at INFO

> log.Printf("[INFO] Policy %s doesn't match %s filter", policy.Name, ic.match)

Then I think this should also be INFO?

Reply:
It will make things too noisy...